client/views/actions/nick.tpl

```diff
@@ -1,3 +1,3 @@
 {{> ../user_name nick=from}}
-is now known as
+{{translate 'client.now_known_as'}}
 {{> ../user_name nick=new_nick}}
```

Comment:
Yeah, the Handlebars template supports passing in options, just need to figure out how :smile: